cli/src/main/resources/hudson/cli/client/Messages_it.properties

```diff
@@ -1,22 +1,22 @@
 CLI.NoSuchFileExists=File non esistente: {0}
-CLI.NoURL=Non sono stati specificati n� -s n� la variabile d''ambiente JENKINS_URL.
-CLI.VersionMismatch=Le versioni non corrispondono. Quest''interfaccia della riga di comando non pu� funzionare con questo server Jenkins.
+CLI.NoURL=Non sono stati specificati n� -s n� la variabile d''ambiente JENKINS_URL.
```

Comment:
is it normal that those characters are shown wrong? Changing then by their unicode codes would avoid troubles. Like in the spanish version.